rasa-ext-vim/src/Rasa/Ext/Vim.hs

```diff
@@ -178,8 +179,19 @@ normal _ = return ()
 
 -- | Move cursors to end of the line
 endOfLine :: BufAction ()
-endOfLine = findNext "\n"
+endOfLine = do 
```

Comment:
This is EXACTLY how I'd have done this 👍 💖 ; really good work! I know it looks a bit messy, but I don't think there's currently a better option. Lucky for us, `asLines` is pretty fast because of how yi-rope is implemented. This is still doing a lot of work though, I do have plans for how we could implement a row-length cache in the future to potentially help with that, but of course that's all just heresay at the moment since we don't yet have any benchmarks.